platform/marvell-armhf/platform.conf

```diff
@@ -10,28 +10,60 @@ fdt_addr=0x1000000
 initrd_addr=0x2000000
 VAR_LOG=512
 
-kernel_fname="/boot/vmlinuz-4.9.0-9-2-armmp"
-initrd_fname="/boot/initrd.img-4.9.0-9-2-armmp"
+kernel_fname="/boot/vmlinuz-4.19.0-9-2-armmp"
+initrd_fname="/boot/initrd.img-4.19.0-9-2-armmp"
 fdt_fname="/boot/armada-385-ET6448M_4G_Nand.dtb"
 
+if [ "$install_env" = "onie" ]; then
+    MACH_FILE="/etc/machine.conf"
+else
+    MACH_FILE="/host/machine.conf"
+fi
+# armhf-marvell_et6448m_52x-r0  - Platform = Et6448M
+# armhf-nokia_ixs7215_52x-r0 - Platform = Nokia IPD6448M
+PLATFORM=`sed -n 's/onie_platform=\(.*\)/\1/p' $MACH_FILE`
```

Comment:
>s/onie_platform=\(.*\)/\1/p [](start = 18, length = 27)

`s/^onie_platform=\(.*\)/\1/p`
Could you add `^` at the beginning of the regex?

Reply:
done